Userland/Libraries/LibGUI/Widget.h

```diff
@@ -50,6 +50,15 @@ enum class VerticalDirection {
     Down
 };
 
+constexpr VerticalDirection KeyCode_to_VerticalDirection(KeyCode const& key)
```

Comment:
This method seems weirdly named, I get `KeyCode` and `VerticalDirection` are types. But using pascal case like this for functions is generally not done.